### PR TITLE
Update docs for v4

### PR DESCRIPTION
**Pull request by robinmetral**
## Purpose

This updates code snippets to the new Emotion 11 packages.

## Approach and changes

Used a keyboard again 🤯 Fantastisch

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: docs/introduction/getting-started.stories.mdx [hunk 1]
```diff
@@ -61,11 +61,11 @@ export default function App() {
 
 ### Configuring the viewport
 
-Finally, make sure that the viewport meta tag includes `viewport-fit=cover`, and that your app has the right padding to render inside [CSS safe areas](https://developer.mozilla.org/en-US/docs/Web/CSS/env()):
+Finally, make sure that the viewport meta tag includes `viewport-fit=cover`, and that your app has the right padding to render inside [CSS safe areas](<https://developer.mozilla.org/en-US/docs/Web/CSS/env()>):
```
Comment (robinmetral) [inline docs/introduction/getting-started.stories.mdx:64]: My linter did this, I guess we can leave it (?)

File: packages/design-tokens/README.md [hunk 2]
```diff
@@ -48,17 +46,19 @@ The package exports a `themePropType` which can be used to check the `theme` pro
...
+function ComponentWithInlineStyles({ theme, label }) {
   return <div style={{ color: theme.colors.p500 }}>{label}</div>;
 }
 
 ComponentWithInlineStyles.propTypes = {
   theme: themePropType.isRequired,
   label: PropTypes.string,
 };
+
+export default function withTheme(ComponentWithInlineStyles);
```
Comment (robinmetral) [inline packages/design-tokens/README.md:61]: Changed this to make the snippet technically correct (the export should be wrapped in `withTheme`) but ideally people should not use the HOC (but the hook) and not use prop-types (but TypeScript) 😛 Maybe we can add TS examples to this in a follow-up PR (similar to what I wrote in the migration guide)

File: docs/introduction/getting-started.stories.mdx (hunk 1 above)
Comment (connor-baer) [reply docs/introduction/getting-started.stories.mdx:64]: What, why? That doesn't look valid. Did you test if it still works?

File: packages/design-tokens/README.md [hunk 3]
```diff
@@ -1,5 +1,3 @@
-<div align="center">
```
Comment (connor-baer) [inline packages/design-tokens/README.md:1]: Why did you remove this (and not the closing tag)?

I prefer the title centered.

**Review by connor-baer — COMMENTED** [review]
Could you also remove the reference to the (non-existent) static CSS docs in `docs/introduction/getting-started.stories.mdx`, please? I forgot to do that a while back.

File: packages/design-tokens/README.md (hunk 3 above)
Comment (robinmetral) [reply packages/design-tokens/README.md:1]: Oops this was a mistake, thanks 🙏 I was gonna verify everything on the deploy preview anyways

File: docs/introduction/getting-started.stories.mdx (hunk 1 above)
Comment (robinmetral) [reply docs/introduction/getting-started.stories.mdx:64]: (in the URL)

Comment (robinmetral) [reply docs/introduction/getting-started.stories.mdx:64]: Probably a way to "escape" weird URL characters

**Review by connor-baer — APPROVED** [review]
